storybook/pages/ExemptionComponentPage.qml

```diff
@@ -0,0 +1,257 @@
+                                            }
+                                        }
+                                    }
+                        }
+
+                        Label {
+                            text: "color"
+                            font.weight: Font.Bold
+                        }
+
+                        TextField {
+                            Layout.fillWidth: true
+                            text: model.color
+                            onTextChanged: model.color = text
+                        }
+
+                        Label {
+                            text: "type"
+                            font.weight: Font.Bold
+                        }
+
+                        Flow {
+                            Layout.fillWidth: true
+
+                            CheckBox {
+                                text: "community"
+                                checked: model.type == 0
+                                onToggled: model.type = 0
+                            }
+                            CheckBox {
+                                text: "user"
+                                checked: model.type == 1
+                                onToggled: model.type = 1
+                            }
+                            CheckBox {
+                                text: "group chat"
+                                checked: model.type == 2
+                                onToggled: model.type = 2
+                            }
+                        }
+
+                        Label {
+                            text: "customized"
+                            font.weight: Font.Bold
+                        }
+
+                        Flow {
+                            Layout.fillWidth: true
+
+                            CheckBox {
+                                text: "customized"
+                                checked: model.customized
+                                onToggled: model.customized = !model.customized
+                            }
+                       }
+
+                        Label {
+                            text: "muteAllMessages"
+                            font.weight: Font.Bold
+                        }
+
+                        TextField {
+                            Layout.fillWidth: true
+                            text: (model.muteAllMessages || "")
+                            onTextChanged: {
+                                if (model.muteAllMessages == "") {
+                                    model.muteAllMessages = false
+                                    return
+                                }
+                                model.muteAllMessages = text
+                            }
+                        }
+
+                        Label {
+                            text: "personalMentions"
+                            font.weight: Font.Bold
+                        }
+
+                        TextField {
```

Comment:
Does it make sense to use here a `RadioButton` with the different allowed options?